bsp/Src/CAN.c

```diff
@@ -139,6 +139,11 @@ CAN_HandleTypeDef* hcan3 = &hcan3_;
 
 // can3 send queue
 static QueueHandle_t can3_send_queue = NULL;
+#ifndef CAN1
+#define CAN_HandleTypeDef void
+#define CAN_TxHeaderTypeDef void
+#define CAN_FilterTypeDef void
+#endif /* CAN1 */
```

Comment:
Why do we have this. If CAN1 isn't defined I don't think the handlers are defined anyway. And ... if they try to use them it might break the code if it's just `void`.